src/scripts/main.js

```diff
@@ -1,3 +1,28 @@
 'use strict';
 
-// write code here
+const listLi = document.querySelectorAll('li');
+const ulItem = document.querySelector('ul');
+
+const sortList = list => {
+  return [...list].sort((a, b) =>
+    +b.dataset.salary.replace(/[$,]/, '')
+    - +a.dataset.salary.replace(/[$,]/, ''));
```

Comment:
```suggestion
    - (+a.dataset.salary.replace(/[$,]/, '')));
```
Використовуй дужки для кращох читабельності.
`- +` виглядає дивно